lib/components/ConfirmationDialog.jsx

```diff
@@ -34,42 +34,59 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/* eslint-disable react/jsx-no-target-blank */
+
 import React from 'react';
 import PropTypes from 'prop-types';
 
 import { Modal, Button, ModalHeader, ModalFooter, ModalBody, ModalTitle } from 'react-bootstrap';
 import Spinner from './Spinner';
 
 const ConfirmationDialog = ({
-    isVisible, isInProgress, title, text, onOk, onCancel, okButtonText, cancelButtonText,
-}) => (
-    <div>
-        <Modal show={isVisible} onHide={onCancel} backdrop={isInProgress ? 'static' : false}>
-            <ModalHeader closeButton={!isInProgress}>
-                <ModalTitle>{title}</ModalTitle>
-            </ModalHeader>
-            <ModalBody>
-                <p>{text}</p>
-            </ModalBody>
-            <ModalFooter>
-                { isInProgress ? <Spinner /> : null }
-                &nbsp;
-                <Button onClick={onOk} disabled={isInProgress}>{okButtonText}</Button>
-                {
-                    onCancel &&
-                        <Button onClick={onCancel} disabled={isInProgress}>
-                            {cancelButtonText}
-                        </Button>
-                }
-            </ModalFooter>
-        </Modal>
-    </div>
+        isVisible,
+        isInProgress,
+        title,
+        text,
+        onOk,
+        onCancel,
+        okButtonText,
+        cancelButtonText,
+        linkText,
+        linkAddress }) => (
```

Comment:
If you end the line with comma and move the closing brace on new line it will be better diff-able and reduces indentation on following block.